Update self-hosted-installation.md (#2069)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/github/self-hosted-installation.md

As this is a self-hosted version, there are some limitations due to security considerations and the fact that we don't have access to your GitHub instance

> - You must include configuration as part of the repository, and you can't configure it via Port's UI/API, please refer to [GitHub Configuration](https://docs.port.io/build-your-software-catalog/sync-data-to-catalog/git/github/?method=github#configuration) for details.

Reviewer: ```suggestion
- The configuration must be included as part of the repository, and cannot be configured via Port's UI/API. Refer to the [GitHub Configuration](https://docs.port.io/build-your-software-catalog/sync-data-to-catalog/git/github/?method=github#configuration) for more details.
```
